docs/quickstart.rst

```diff
@@ -1,4 +1,83 @@
 .. _quickstart:
 
 Quickstart
-==========
\ No newline at end of file
+==========
+
+Installation
+------------
+
+.. raw:: html
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
+                <div class="card install-card shadow w-100">
+                <div class="card-header">
+                    Working with conda?
+                </div>
+                <div class="card-body">
+                    <p class="card-text">
+
+Fairlearn is on `conda-forge <https://anaconda.org/conda-forge/fairlearn>`_:
+
+.. raw:: html
+
+                    </p>
+                </div>
+                <div class="card-footer text-muted">
+
+.. code-block:: bash
+
+    conda install -c conda-forge fairlearn 
+
+.. raw:: html
+
+                </div>
+                </div>
+            </div>
+            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
+                <div class="card install-card shadow w-100">
+                <div class="card-header">
+                    Prefer pip?
+                </div>
+                <div class="card-body">
+                    <p class="card-text">
+
+Fairlearn can be installed via pip from `PyPI <https://pypi.org/project/fairlearn>`__.
+
+.. raw:: html
+
+                    </p>
+                </div>
+                <div class="card-footer text-muted">
+
+.. code-block:: bash
+
+   pip install fairlearn
+
+.. raw:: html
+
+                </div>
+                </div>
+            </div>
+            <div class="col-12 d-flex install-block">
+                <div class="card install-card shadow w-100">
+                <div class="card-header">
+                    In-depth instructions?
+                </div>
+                <div class="card-body">
+                    <p class="card-text">Installing a specific version?
+                      Installing from source?
+                      Check the advanced installation page.</p>
+
+.. container:: custom-button
+
+    :ref:`Learn more <advanced_install>`
+
+.. raw:: html
+
+                </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
Can you copy the content of the section https://github.com/fairlearn/fairlearn#overview-of-fairlearn to right after the installation instructions?

Reply:
That raises the question of whether to remove them from the README itself. I strongly dislike duplication, but the README needs to have a basic summary and then link to the webpage for the rest. I'll copy this over for now, but before we complete the PR we should address this.

Reply:
It's also a little more detailed than I'd like for the quickstart. Some of that content belongs elsewhere....

There's also the question about where 'vision' and 'roadmap' and such things would go. Vision is such an important topic that it needs to be easily discoverable, so we can't hide it somewhere as a subsection of community or something like that.

Reply:
I thought about this quite a bit now, and feel like the quickstart should be short so that it's actually a quick start. There's the short section on the two components, and then it dives into an example. For the more comprehensive description of the dashboard, and the table of algorithms you now have to go to the user guides. LMK what you think.

Reply:
Sounds good. I'll add a bit more on metrics in a future pass.